board/app/controllers/comment_controller.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+class CommentController extends AppController 
+{
+
+const WRITE_COMMENT = 'write';
+const WRITE_COMMENT_END = 'write_end';
+const MAX_USER_IN_NEWSFEED = 1;
+
+    public function write() 
+    {
+        $thread_id = Param::get('thread_id');
+        $thread = Thread::get($thread_id);
+        $current_page = Param::get(PAGE_NEXT, self::WRITE_COMMENT);
+    
+        switch($current_page) { 
+            case self::WRITE_COMMENT:
+                break;  
+                
+            case self::WRITE_COMMENT_END:
+                $params = array(
+                'body' => Param::get('body'),
+                'username' => $_SESSION['username'],
+                'user_id' => $_SESSION['user_id']
```

Comment:
Done po. (Applied to all). Thank you :bow:
--> https://github.com/maragatchalian/FoodFacts/commit/3d90ceb5bfe86bf2a5e3dfaee3b5cbce5df1ee37